examples/grpc-server-with-prometheus/server/server.go

```diff
@@ -72,8 +72,7 @@ func main() {
 	// Register your service.
 	pb.RegisterDemoServiceServer(grpcServer, demoServer)
 
-	// Initialize all metrics.
-	grpcMetrics.InitializeMetrics(grpcServer)
```

Comment:
yes, comment in on place, but it does remove publishing initial metrics

Reply:
as I understood the lib initialises all metrics with 0 value (method_name*N metrics). Now this initialisation is omitted. What will be in case we try to gather metrics just after initialisation (there were no gRPC API calls) ?

Reply:
Yes, they will not be listed. It is not critical for us. But having thousands empty API services stat can cause issues. Because we need to store that for each user.